rust/frontend/src/auth/mod.rs

```diff
@@ -39,6 +40,7 @@ impl Display for AuthzAction {
             AuthzAction::Reset => write!(f, "system:reset"),
             AuthzAction::CreateTenant => write!(f, "tenant:create_tenant"),
             AuthzAction::GetTenant => write!(f, "tenant:get_tenant"),
+            AuthzAction::UpdateTenant => write!(f, "tenant:update_tenant"),
```

Comment:
Are there any rollout/migration considerations with this call since MP has a copy of these actions?

Reply:
No, the MP pulls from OSS so it will receive the new enum when the changes go out. There is no usage that would break with the new value (e.g. a `match`).